src/style.css

```
@@ -0,0 +1,140 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  margin-right: 3px;
+  margin-left: 3px;
+  margin-top: 32px;
+}
+
+.product-card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  flex-grow: 1;
+  width: 100%;
+  margin-bottom: 4px;
+  margin-top: 0;
+}
+
+.product-card__article {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0 0 16px;
+}
+
+.product-card__star {
+  height: 16px;
+  width: 16px;
+  background: url(./images/star.svg) no-repeat;
+}
+
+.product-card__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.product-card__stars {
+  display: flex;
+}
+
+.product-card__stars--0 .product-card__star:nth-child(-n) {
+  background: url(./images/star-active.svg) no-repeat;
+}
+
+.product-card__stars--1 .product-card__star:nth-child(-n+1) {
+  background: url(./images/star-active.svg) no-repeat;
+}
+
+.product-card__stars--2 .product-card__star:nth-child(-n+2) {
+  background: url(./images/star-active.svg) no-repeat;
+}
+
+.product-card__stars--3 .product-card__star:nth-child(-n+3) {
+  background: url(./images/star-active.svg) no-repeat;
+}
+
+.product-card__stars--4 .product-card__star:nth-child(-n+4) {
+  background: url(./images/star-active.svg) no-repeat;
+}
+
+.product-card__stars--5 .product-card__star:nth-child(-n+5) {
+  background: url(./images/star-active.svg) no-repeat;
+}
```

```suggestion
.product-card__stars--1 .product-card__star:nth-child(-n+1),
.product-card__stars--2 .product-card__star:nth-child(-n+2),
.product-card__stars--3 .product-card__star:nth-child(-n+3),
.product-card__stars--4 .product-card__star:nth-child(-n+4),
.product-card__stars--5 .product-card__star:nth-child(-n+5) {
  background: url(./images/star-active.svg) no-repeat;
}
```
Не повторюй стилі, а самий перший взагалі не потрібен